packages/agent/conf/examples/oid4vci_metadata/knb_babylon.json

```diff
@@ -0,0 +1,168 @@
+{
+  "correlationId": "https://ssi.dutchblockchaincoalition.org/issuer",
+  "overwriteExisting": true,
+  "metadata": {
+    "credential_issuer": "https://ssi.dutchblockchaincoalition.org/issuer",
+    "credential_endpoint": "https://ssi.dutchblockchaincoalition.org/issuer/credentials",
+    "display": [
+      {
+        "name": "Babylon Bank",
+        "description": "Babylon Bank Issuer"
+      }
+    ],
+    "credentials_supported": [
+      {
+        "display": [
+          {
+            "name": "Bank account credential",
+            "description": "Bank account credential",
+            "background_color": "rgba(0, 0, 0, 0.2)",
+            "text_color": "#FBFBFB",
+            "logo": {
+              "url": "https://${issuer}/assets/images/icons/Babylon_bank_logo.png",
```

Comment:
AFAIK there is no var support here. Just upload them to imgbb like the other images

Reply:
Fixed